app/models/concerns/filterable.rb

```diff
@@ -13,13 +13,13 @@ def filter_scope(name, *args)
       filter_scopes << name
     end
 
-    def filter(filtering_params)
+    def filter_by(filtering_params)
       results = all
       filtering_params.each do |filter_scope, filter_value|
-        filter_value = filter_value.reject(&:blank?) if filter_value.is_a?(Array)
```

Comment:
2. Positive inflection

Usually it's easier to follow `if clause` vs `unless !clause`. Can be debatable though. Curious what was the reason for the following change in 048e67069d5b6fc4cdcae7276af2d4cf16149dde:
```ruby
- results = results.public_send(filter_scope, filter_value) if filter_value.present?
+ results = results.public_send(filter_scope, filter_value) unless filter_value.blank?
```